Dockerfile

```diff
@@ -14,13 +13,24 @@ RUN apt-get update && apt-get install ca-certificates && \
     apt-get update && \
     apt-get install -y yarn
 
-WORKDIR /data/
+RUN echo ${VERSION} > /version.txt
+
+WORKDIR /src/
+#get dependencies
+COPY package.json .
+COPY yarn.lock .
+COPY static .
```

Comment:
static might contain many other files,  including binary builds, .js.map files.
And should be excluded explicitly.

Reply:
This requires the currently checked-out repo to be clean.

Reply:
@andelf i included it because of the yarn.lock file in there and it is the only file checked in, i'll make it explicit and only include the lock file

Reply:
btw, why to exclude this step in separate build step is to profit of layer caching, i just couldn't find out on how to fetch the deps.edn dependencies.